algorithms/kernel/low_order_moments/oneapi/low_order_moments_kernel_batch_oneapi.h

```diff
@@ -135,6 +135,7 @@ class LowOrderMomentsBatchTaskOneAPI : public TaskInfoBatch<algorithmFPType, sco
     unsigned int nFeatures;
 
     const unsigned int maxWorkItemsPerGroup = 256;
```

Comment:
Number of work items per work group depends on hardware capabilities, I would recommend to not hardcode it, but get it from device info.

Reply:
Yes, it will be rewritten during transition to new interfaces.